generator/AWSPSGeneratorLib/HelpMaterials/Examples/ELB2/Add-ELB2ListenerCertificate.xml

```diff
@@ -0,0 +1,16 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--Examples for the AWS PowerShell CmdLets-->
+<examples>
+  <example>
+    <code>$newCert = New-Object -TypeName Amazon.ElasticLoadBalancingV2.Model.Certificate
+
+$newCert.CertificateArn = 'arn:aws:acm:us-east-1:123456789012:certificate/19478bd5-491d-47d4-b1d7-5217feba1d97'
+
+Add-ELB2ListenerCertificate -ListenerArn 'arn:aws:elasticloadbalancing:us-east-1:123456789012:listener/app/test-alb/3651b4394dd9a24f/3873f123b98f7618' -Certificate $newCert -Region us-east-1
```

Comment:
This can be simplified with `-Certificate @{CertificateArn = 'arn:aws:acm:us-east-1:123456789012:certificate/19478bd5-491d-47d4-b1d7-5217feba1d97'}`